deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -550,17 +554,17 @@ def loop_logger(instancestates: Dict[str, Any], terminateoncompletion: bool) ->
             rootLogger.info("Instances")
             rootLogger.info("-"*80)
             for instance in instancestate_map.keys():
-                rootLogger.info("""Hostname/IP:{:>15} | Terminated: {}""".format(instance, truefalsecolor[instancestate_map[instance]]))
+                rootLogger.info("""Hostname/IP:{:>{}} | Terminated: {}""".format(instance, longestinst, truefalsecolor[instancestate_map[instance]]))
```

Comment:
Did you test this?  I vaguely remember having to do what's described in https://stackoverflow.com/questions/46573063/python-3-nesting-string-format when wanting to nest string formatting specifiers in python.

Reply:
I tested this on a Python3 online IDE. I'll test this on my AMI when I get a chance.

Reply:
I can confirm that this worked with the following code snippet in `firesim` top-level file.

```
i = 1
j = 2
print("""{:>{}}""".format(i, 2))
print("""{:>{}}""".format(j, 4))
```

Gave:

```
 1
  22
```